crow/crow_modules/randomENGpy2.i

```diff
@@ -0,0 +1,14 @@
+%module randomENGpy2
```

Comment:
would it be possible to leave a description here?

Reply:
I would prefer to create a guide to do what I just did for the RNG. Can you open an issue and assign to me?